docs/design/gatewayapi-translator.md

```diff
@@ -1,35 +1,38 @@
 # Gateway API Translator Design
 
+The Gateway API translates external resources, e.g. GatewayClass, from the configured Provider's Resource Watcher to the
```

Comment:
should we just stick to `Provider` and rm `Resource Watcher` ?

Reply:
Ack.